File: articles/conceptual-modeling-in-complex-systems/cpp/complex_system_model_metrics.cpp
#include <iostream>
#include <iomanip>

int main() {
    double components = 10.0;
    double relationships = 12.0;

    std::cout << std::fixed << std::setprecision(3);
    std::cout << "Metadata coverage: " << 8.0 / components << "\n";
    std::cout << "Uncertainty context coverage: " << 8.0 / components << "\n";
    std::cout << "Relationship traceability: " << 10.0 / relationships << "\n";
    std::cout << "Feedback edge share: " << 8.0 / relationships << "\n";
    std::cout << "Underspecified relationship risk: " << 1.0 / relationships << "\n";

    return 0;
}

File: articles/conceptual-modeling-in-complex-systems/fortran/complex_system_model_metrics.f90
program complex_system_model_metrics
  implicit none

  real :: components
  real :: relationships

  components = 10.0
  relationships = 12.0

  print *, "Complex Systems Conceptual Model Metrics"
  print *, "Metadata coverage: ", 8.0 / components
  print *, "Uncertainty context coverage: ", 8.0 / components
  print *, "Relationship traceability: ", 10.0 / relationships
  print *, "Feedback edge share: ", 8.0 / relationships
  print *, "Underspecified relationship risk: ", 1.0 / relationships
end program complex_system_model_metrics
